### ISSUE TITLE
Missing files in package's template folder

### ISSUE THREAD

**Issue by zimme** [vadimdemedes/create-pastel-app#2]
It seems that files are missing from this package's template folder.
```
➜  create-project-pastel npm init pastel-app
npx: installed 264 in 10.048s

  ✖ Copy files
    → ENOENT: no such file or directory, open '/Users/zimme/.npm/_npx/20561/lib/node_modules/create-pastel-app/template/package.json'
    Install dependencies
    Build application
    Link executable
Error: ENOENT: no such file or directory, open '/Users/zimme/.npm/_npx/20561/lib/node_modules/create-pastel-app/template/package.json'
➜  create-project-pastel ls -la ~/.npm/_npx/19419/lib/node_modules/create-pastel-app/template
total 8
drwxr-xr-x  3 zimme  staff   96 May  1 19:33 .
drwxr-xr-x  9 zimme  staff  288 May  1 19:33 ..
-rw-r--r--  1 zimme  staff  473 Oct 26  1985 readme.md
➜  create-project-pastel
```
I run into the same problem when using `npx create-pastel-app`

**Comment by vadimdemedes** [comment]
Ah, I forgot that npm ignores nested `package.json` files. Will push up a fix shortly.

**Comment by vadimdemedes** [comment]
Fixed in `1.0.2`!
